src/routes/transactions/mappers/transfers/swap-transfer-info.mapper.ts

```diff
@@ -61,7 +63,11 @@ export class SwapTransferInfoMapper {
 
     // TODO: Refactor with confirmation view, swaps and TWAPs
     if (!this.swapOrderHelper.isAppAllowed(order)) {
-      throw new Error(`Unsupported App: ${order.fullAppData?.appCode}`);
+      this.loggingService.warn(
+        `Unsupported App: ${order.fullAppData?.appCode}`,
+      );
+      // Don't throw in order to fallback to "standard" transfer mapping
+      return null;
```

Comment:
Nit: while I see `null` returns in many other places related to transactions/swap/twaps mappings to indicate a failure, I also think we should refactor the `null` returns and change them for actual errors and do proper error handling in the upper layers/caller functions, wdyt? (This is just a general comment, I'm totally fine with this PR 🙂)

Reply:
I agree. There is a common pattern with the finding then mapping of swap-related transactions that needs to be refactored in general. I think it would also solve this.

Reply:
"Proper error handling" has been included in https://github.com/safe-global/safe-client-gateway/pull/1733, but it should definitely be refactored as mentioned.